doc(Chrome60-SW requirements): fix notes and info

NathalieC opened the pull request:
* Reorder notes
* Add "or" to explain what browser to use to execute the fixing procedure

NathalieC committed:
doc(Chrome60-SW requirements): fix notes and info

Reordered notes
Add "or" to explain what browser to use to execute the fixing procedure
Removed BPM after Bonita when needed in the new content

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/bonitasoft/bonita-doc?pullRequest=399) <br/>All committers have signed the CLA.

NathalieC committed:
doc(Chrome60-HW requirements): add BPM

For 7.5, "Bonita BPM" is the official name
I'll check in 7.6 doc how it behaves.

educhastenier commented on md/hardware-and-software-requirements.md:
```diff
@@ -44,20 +44,22 @@ Software required for Bonita BPM Platform (Bonita BPM Engine and Bonita BPM Port
 :::
 
-1. Log in the Bonita Portal as Administrator.
+1. Log in to Bonita BPM Portal as Administrator.
```
Will need to remove all those `BPM` when merging to 7.6 => need a task, not to forget

abirembaut commented:
LGTM but same comment (Bonita BPM)

NathalieC commented:
Sure. When will you merge this page into 7.6? Where is the best place to create this task?
